tests/Doctrine/Migrations/Tests/Provider/OrmSchemaProviderTest.php

```diff
@@ -37,13 +36,13 @@ public function testCreateSchemaFetchesMetadataFromEntityManager(): void
         }
     }
 
-    public function testEntityManagerWithoutMetadataCausesError(): void
+    public function testEntityManagerWithoutMetadata(): void
```

Comment:
I think this warrants an explanation alongside the test.

```suggestion
    /**
     * It should be OK to use migrations to manage tables not managed by
     * the ORM.
     *
     * @doesNotPerformAssertions
     */
    public function testEntityManagerWithoutMetadataIsFine(): void
```

Reply:
updated, thanks